SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2260,7 +2260,12 @@ void USpatialReceiver::ResolveObjectReferences(FRepLayout& RepLayout, UObject* R
 
 		if (ObjectReferences.Array)
 		{
-			check(Property->IsA<UArrayProperty>());
+			UArrayProperty* ArrayProperty = Cast<UArrayProperty>(Property);
+			if (ArrayProperty == nullptr)
+			{
+				UE_LOG(LogSpatialReceiver, Error, TEXT("Failed to cast an array of properties to UArrayProperty"));
```

Comment:
Can you output the property name also? And perhaps the target object?

Reply:
Ignoring this request, since the `if statement` will be swapped with `check`.